spec/haml_lint/ruby_extractor_spec.rb

```diff
@@ -224,12 +224,51 @@
       HAML
 
       its(:source) { should == normalize_indent(<<-RUBY).rstrip }
-        {}.merge(one: 1, two: 2, 'three' => 3)
+        {}.merge(one: 1,
+        two: 2,
+        'three' => 3)
         _haml_lint_puts_0 # tag
         _haml_lint_puts_1 # tag/
       RUBY
 
-      its(:source_map) { should == { 1 => 1, 2 => 1, 3 => 1 } }
+      its(:source_map) { should == { 1 => 1, 2 => 1, 3 => 1, 4 => 1, 5 => 1 } }
+    end
+
+    # Multiline attributes were introduced in 5.2.1
+    if Haml::VERSION >= '5.2.1'
```

Comment:
Just to confirm: should this conditional be in the linter itself, not in the tests? Otherwise anyone running the linter with a version of HAML would see the test failures this is attempting to correct, or am I misunderstanding the conditions where this occurs?

Reply:
I think this is the same case like in `space_inside_hash_attributes_spec.rb` – Any case where this linter could activate would already be invalid in old versions of HAML. But if you‘d rather have it moved, I can do that!